ChangeLog.md

```diff
@@ -1,3 +1,193 @@
+## 3.2.0 - December 2019
```

Comment:
Please add one section below
```
### General
* Update references in .psd1 to use relative path
```
and remove this general changes below for all modules